src/vs/workbench/contrib/welcome/defaultExtensionIcon.svg

```diff
@@ -0,0 +1,25 @@
+
+<svg id="a310ce98-1fce-406a-8038-d49027539a51" data-name="icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="18" height="18" viewBox="0 0 18 18">
```

Comment:
If this is still needed it should go into the sql part of the tree since I don't see VS Code having their own version

Reply:
resolved